pulp/apis/highs_api.py

```diff
@@ -135,20 +135,41 @@ def actualSolve(self, lp):
                 option += f"={next(options)}"
 
             # identify cli options by a leading dash (-) and treat other options as file options
-            if option.starts_with("-"):
+            if option.startswith("-"):
                 command.append(option)
             else:
                 file_options.append(option)
 
         with open(tmpOptions, "w") as options_file:
             options_file.write("\n".join(file_options))
-        process = subprocess.run(command, stdout=sys.stdout, stderr=sys.stderr)
+
+        if self.msg:
+            pipe = None
+        else:
+            pipe = open(os.devnull, "w")
+
+        lp_status = None
+
+        # 		The following method of starting the subprocess causes exception UnsupportedOperation("fileno") in iostream.py in Windows
+        #        process = subprocess.run(command, stdout=sys.stdout, stderr=sys.stderr, universal_newlines=True)
+
+        with subprocess.Popen(
+            command,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT,
+            universal_newlines=True,
+        ) as proc, open(tmpLog, "w") as log_file:
+            for line in proc.stdout:
+                if self.msg:
+                    sys.__stdout__.write(line)
+                log_file.write(line)
 
```

Comment:
Check how it's done in the solve method of `coin_api.COIN_CMD class`. Ideally we would like to have the same style:

```
if self.msg:
    pipe = None
else:
    pipe = open(os.devnull, "w")
process = subprocess.Popen(args, stdout=pipe, stderr=pipe, stdin=devnull)
if process.wait() == -1:
    if pipe:
        pipe.close()
    raise PulpSolverError(
        "Pulp: Error while trying to execute, use msg=True for more details"
        + self.path
    )
if pipe:
    pipe.close()